sdk/tests/deqp/functional/gles3/es3fNegativeBufferApiTests.js

```diff
@@ -564,22 +564,23 @@ goog.scope(function() {
                 gl.checkFramebufferStatus(gl.FRAMEBUFFER);
                 this.expectError(gl.NO_ERROR);
 
-                bufferedLogToConsole('gl.INVALID_ENUM is generated if one of the values in bufs is not an accepted value.');
-                gl.drawBuffers(values.splice(2, 2));
-                this.expectError(gl.INVALID_ENUM);
+                bufferedLogToConsole('gl.INVALID_OPERATION is generated if the GL is bound to framebuffer object and one of the values in bufs is not an accepted value.');
+                gl.drawBuffers(values.slice(0, 2));
+                this.expectError(gl.INVALID_OPERATION);
```

Comment:
Agree that the ES 3.0.4 spec is unclear for this error case, but the manual page is pretty clear that INVALID_ENUM should be generated here. The reason it's generated is that DEPTH_ATTACHMENT is not a valid buffer to name in the drawBuffers call.

There is a bug here though. It should be calling:
  gl.drawBuffers(values.slice(2, 4))
not "splice".